### PR TITLE
Completed css-two-column-layout exercise. Bryan Valdovinos

### PR DESCRIPTION
**Pull request by bryanvaldovinos**
Screen Shot 2022-05-13 at 5 18 26 PM

### CODE REVIEW

File: css-two-column-layout/styles.css [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+  flex-direction: column;
+}
+
+.smaller {
+  width: 460px;
+}
+
+.row2 {
+  font-family: Karla, sans-serif;
+}
```
Comment (codingcodymiller) [inline css-two-column-layout/styles.css:63]: 1. Do not number your classes, it gives no insight into their meaning or usage.
2. You should just make this a utility class to apply font family.

File: css-two-column-layout/styles.css [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+.pic {
+  position: relative;
+  bottom: 30px;
+}
+
+.sixtysix {
+  width: 66%;
+}
```
Comment (codingcodymiller) [inline css-two-column-layout/styles.css:93]: Why not `column-two-thirds`?

File: css-two-column-layout/styles.css [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+
+.thirtythree {
+  width: 33%;
+}
```
Comment (codingcodymiller) [inline css-two-column-layout/styles.css:97]: Why not `column-third`?

File: css-two-column-layout/styles.css [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+
+.smaller {
+  width: 460px;
+}
+
+.row2 {
+  font-family: Karla, sans-serif;
+}
+
+.shrinktext {
```
Comment (codingcodymiller) [inline css-two-column-layout/styles.css:65]: If your class name includes multiple words, you should be using `kebab-case`

File: css-two-column-layout/styles.css [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+  line-height: 1.25rem;
+  font-size: 70%;
+}
+
+.alignleft {
```
Comment (codingcodymiller) [inline css-two-column-layout/styles.css:70]: If your class name includes multiple words, you should be using `kebab-case`

File: css-two-column-layout/styles.css [hunk 6]
```diff
@@ -0,0 +1,118 @@
...
+  text-align: left;
+}
+
+.spacebelow {
```
Comment (codingcodymiller) [inline css-two-column-layout/styles.css:74]: If your class name includes multiple words, you should be using `kebab-case`

File: css-two-column-layout/styles.css [hunk 7]
```diff
@@ -0,0 +1,118 @@
...
+  margin-bottom: 10px;
+}
+
+.smallertitle {
```
Comment (codingcodymiller) [inline css-two-column-layout/styles.css:78]: If your class name includes multiple words, you should be using `kebab-case`

File: css-two-column-layout/styles.css [hunk 8]
```diff
@@ -0,0 +1,118 @@
...
+  font-size: 80%;
+}
+
+.marginbottom {
```
Comment (codingcodymiller) [inline css-two-column-layout/styles.css:82]: If your class name includes multiple words, you should be using `kebab-case`

File: css-two-column-layout/styles.css [hunk 9]
```diff
@@ -0,0 +1,118 @@
...
+.thirtythree {
+  width: 33%;
+}
+
+img {
+  width: 100px;
+  height: 100px;
+}
+
+.spaceabove {
```
Comment (codingcodymiller) [inline css-two-column-layout/styles.css:104]: If your class name includes multiple words, you should be using `kebab-case`

File: css-two-column-layout/styles.css [hunk 10]
```diff
@@ -0,0 +1,118 @@
...
+  margin-top: 50px;
+}
+
+.spaceleft {
```
Comment (codingcodymiller) [inline css-two-column-layout/styles.css:108]: If your class name includes multiple words, you should be using `kebab-case`

File: css-two-column-layout/styles.css [hunk 11]
```diff
@@ -0,0 +1,118 @@
...
+  margin-left: 40px;
+}
+
+.spaceright {
```
Comment (codingcodymiller) [inline css-two-column-layout/styles.css:112]: If your class name includes multiple words, you should be using `kebab-case`

File: css-two-column-layout/styles.css [hunk 12]
```diff
@@ -0,0 +1,118 @@
...
+  margin-right: 40px;
+}
+
+.margintop {
```
Comment (codingcodymiller) [inline css-two-column-layout/styles.css:116]: If your class name includes multiple words, you should be using `kebab-case`

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by bryanvaldovinos** [comment]
Class Names changed!
Screen Shot 2022-05-15 at 7 14 15 PM

File: css-two-column-layout/styles.css [hunk 13]
```diff
@@ -0,0 +1,118 @@
...
+
+.row {
+  display: flex;
+}
+
+.gray-font {
+  color: #a1a1a1;
+}
+
+.column {
```
Comment (codingcodymiller) [inline css-two-column-layout/styles.css:14]: You should make this `column-full`

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
In addition to my other comment, I think you may have some rulesets that you never ended up using still in your stylesheet, please verify that you have only included styling that is actually necessary

**Review by codingcodymiller — APPROVED** [review]
Nice work!!

**Pull request merged**